components/retro-go/rg_system.c

```diff
@@ -416,6 +416,7 @@ rg_app_t *rg_system_init(int sampleRate, const rg_handlers_t *handlers, const rg
     if (enterRecoveryMode)
     {
         rg_display_init();
+        rg_settings_init();
```

Comment:
I really don't want settings to be loaded during recovery, because settings could be the cause of the previous crash.

But I also understand why a localized menu would be desirable so we'll have to figure something out.

Reply:
yeah I agree, it crash without an sdcard, so not ideal. Maybe we can use a byte in the ESP32 flash to store the current language ?

Reply:
Indeed I suppose we could store some settings in NVS. But it has some downsides too...

For now I would suggest to comment the added `rg_settings_init();` and put a `// FIXME: Find a way to get the user's language without loading all settings` (or something along those lines).

It's not that big of a deal of the recovery menu is in English (or French, if your target defaults to french) and let's not let this small issue block progress on the PR.